src/Morphir/Cadl/Backend.elm

```diff
@@ -26,24 +27,29 @@ mapDistribution : Options -> Distribution -> Result Errors FileMap
 mapDistribution opt distro =
     case distro of
         Library packageName _ packageDef ->
-            mapPackageDefinition packageDef
-                |> Result.map (prettyPrint packageName)
-                |> Result.map (Dict.singleton ( [], Path.toString IRName.toTitleCase "." packageName ++ ".cadl" ))
+            let
+                shouldImportSDK : Bool
+                shouldImportSDK =
+                    packageDef.modules
+                        |> Dict.toList
+                        |> List.map (Tuple.second >> .value >> Module.collectTypeReferences)
+                        |> List.foldl Set.union Set.empty
+                        |> Set.member ( [ [ "morphir" ], [ "s", "d", "k" ] ], [ [ "decimal" ] ], [ "decimal" ] )
 
+                imports : List ImportDeclaration
+                imports =
+                    if shouldImportSDK then
+                        [ Absolute "./sdk.cadl" ]
```

Comment:
This is confusing. You call it absolute but this is a relative path. Can you add some docs to the type declaration and make sure this is used consistently?